<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addinhashdialog</class>
 <widget class="QDialog" name="addinhashdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="passportLabel">
           <property name="text">
            <string>Номер SIM-карты</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Тариф</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="yearLabel">
           <property name="text">
            <string>Год выпуска</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="adressLabel">
           <property name="text">
            <string>Признак наличия</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="simNumberEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="rateEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="yearEdit"/>
         </item>
         <item>
          <widget class="QComboBox" name="isThereComboBox">
           <item>
            <property name="text">
             <string>Нет </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Да</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="doneButton">
         <property name="text">
          <string>Ок</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
